src/core/ad.jl

```diff
@@ -39,14 +38,15 @@ ADBackend() = ADBackend(ADBACKEND[])
 ADBackend(T::Symbol) = ADBackend(Val(T))
 
 ADBackend(::Val{:forward_diff}) = ForwardDiffAD{CHUNKSIZE[]}
-ADBackend(::Val) = TrackerAD
+ADBackend(::Val{:reverse_diff}) = TrackerAD
+ADBackend(::Val) = error("The requested AD backend is not available. Make sure to load all required packages.")
 
 """
-getADtype(alg)
+getADbackend(alg)
 
-Finds the autodifferentiation type of the algorithm `alg`.
+Finds the autodifferentiation backend of the algorithm `alg`.
```

Comment:
According to the [Julia documentation](https://docs.julialang.org/en/latest/manual/documentation/#Documentation-1), one should use the imperative form:
```suggestion
Find the autodifferentiation backend of the algorithm `alg`.
```